beacon/dkg.go

```diff
@@ -92,6 +98,7 @@ type DistributedKeyGeneration struct {
 	stateDuration   int64
 	aeonKeys        *aeonDetails
 	onFailState     func(int64)
+	enableRecovery  bool
```

Comment:
Is this so we can turn off recovery for tests?

Reply:
Yup